galasa-simbank-tests/dev.galasa.simbank.tests/src/main/java/dev/galasa/simbank/tests/WebAppIntegrationTest.java

```diff
@@ -0,0 +1,148 @@
+package dev.galasa.simbank.tests;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.net.InetSocketAddress;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.logging.Log;
+import org.assertj.core.api.Fail;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import dev.galasa.Test;
+import dev.galasa.artifact.BundleResources;
+import dev.galasa.artifact.IBundleResources;
+import dev.galasa.artifact.TestBundleResourceException;
+import dev.galasa.core.manager.CoreManager;
+import dev.galasa.core.manager.ICoreManager;
+import dev.galasa.core.manager.Logger;
+import dev.galasa.zos.IZosImage;
+import dev.galasa.zos.ZosImage;
+import dev.galasa.zos3270.FieldNotFoundException;
+import dev.galasa.zos3270.ITerminal;
+import dev.galasa.zos3270.KeyboardLockedException;
+import dev.galasa.zos3270.TerminalInterruptedException;
+import dev.galasa.zos3270.TextNotFoundException;
+import dev.galasa.zos3270.TimeoutException;
+import dev.galasa.zos3270.Zos3270Terminal;
+import dev.galasa.zos3270.spi.NetworkException;
+import dev.galasa.zosbatch.IZosBatch;
+import dev.galasa.zosbatch.IZosBatchJob;
+import dev.galasa.zosbatch.IZosBatchJobname;
+import dev.galasa.zosbatch.ZosBatch;
+import dev.galasa.zosbatch.ZosBatchException;
+import dev.galasa.zosbatch.ZosBatchJobname;
+import dev.galasa.selenium.ISeleniumManager;
+import dev.galasa.selenium.IWebPage;
+import dev.galasa.selenium.SeleniumManager;
+import dev.galasa.selenium.SeleniumManagerException;
+import dev.galasa.docker.DockerContainer;
+import dev.galasa.docker.DockerManagerException;
+import dev.galasa.docker.IDockerContainer;
+
+@Test
+public class WebAppIntegrationTest {
+
+	@DockerContainer(image = "maven-web-app", dockerContainerTag = "a", start = true)
+    public IDockerContainer container;
+	
+	@ZosImage(imageTag = "SIMBANK")
+	public IZosImage        image;
+
+	@Zos3270Terminal(imageTag = "SIMBANK")
+	public ITerminal        terminal;
+	
+	@ZosBatch(imageTag="SIMBANK")
+    public IZosBatch zosBatch;
+    
+    @ZosBatchJobname(imageTag="SIMBANK")
+	public IZosBatchJobname zosBatchJobname;
+	
+	@CoreManager
+    public ICoreManager     coreManager;
+	
+	@BundleResources
+	public IBundleResources resources; 
+
+	@SeleniumManager
+    public ISeleniumManager seleniumManager;
+
+	@Logger
+	public Log logger;
```

Comment:
Might be nice to log what we are doing as the test progresses.  when we get an error then log a message to describe the problem